utils/build/docker/otelcol-config.yaml

```diff
@@ -0,0 +1,30 @@
+receivers:
+  otlp:
+    protocols:
+      http:
+        endpoint: 0.0.0.0:4318
+
+exporters:
+  datadog:
+    traces:
+      span_name_as_resource_name: true
+    api:
+      key: ${DD_API_KEY}
+      site: ${DD_SITE}
+      fail_on_invalid_key: true
+  logging:
+    verbosity: detailed
```

Comment:
is this for debugging ?

Reply:
Yes the logs will be stored in `logs_otel_tracing_e2e/docker/collector` for debugging